### PR TITLE
[ROX-11918]: Add custom route to central configuration

### PR DESCRIPTION
**Pull request by andykrohg**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/ROX-11918

Link to docs preview:
https://github.com/andykrohg/openshift-docs/blob/rhacs-docs/modules/central-configuration-options-operator.adoc

Additional information:

Pending merge of PR in stackrox: https://github.com/stackrox/stackrox/pull/2621

### CODE REVIEW

File: modules/central-configuration-options-operator.adoc [hunk 1]
```diff
@@ -40,6 +40,9 @@ Otherwise, you must reinstall the custom resource to log back in.
 |`central.exposure.route.enabled`
 |Set this to `true` to expose Central through an OpenShift route. The default value is `false`.
 
+|`central.exposure.route.host`
+|Specify a custom host name to use for Central's route. Leave this blank to accept the default value from OpenShift.
```
Comment (porridge) [inline modules/central-configuration-options-operator.adoc:44]:
```suggestion
|Specify a custom host name to use for Central's route. Leave this unset to accept the default value from OpenShift.
```
Not sure how a route would behave if host is set to an empty string (as opposed to unset), but better safe than sorry.

**Review by porridge — CHANGES_REQUESTED** [review]

**Comment by andykrohg** [comment]
Ready for merge whenever this makes sense! Stackrox PR was merged @gaurav-nelson  : https://github.com/stackrox/stackrox/pull/2621

**Comment by andykrohg** [comment]
Looks like this change is live in ACS 3.72. Is this ready for merge?

File: modules/central-configuration-options-operator.adoc [hunk 2]
```diff
@@ -40,6 +40,9 @@ Otherwise, you must reinstall the custom resource to log back in.
 |`central.exposure.route.enabled`
 |Set this to `true` to expose Central through an OpenShift route. The default value is `false`.
 
+|`central.exposure.route.host`
+|Specify a custom host name to use for Central's route. Leave this unset to accept the default value from OpenShift.
```
Comment (gaurav-nelson) [inline modules/central-configuration-options-operator.adoc:44]:
```suggestion
|Specify a custom hostname to use for Central's route. Leave this unset to accept the default value that {ocp} provides.
```

**Comment by michaelryanmcneill** [comment]
Make sure to rebase your commits!

**Comment by andykrohg** [comment]
> Make sure to rebase your commits!

All set!

**Comment by michaelryanmcneill** [comment]
Per Slack, removing `peer-review-required`, adding `peer-review-done`, and adding `merge-review-needed`. 

Merge team, please cherry pick into: `rhacs-docs-3.72`

/label peer-review-done
/label merge-review-needed
/remove-label peer-review-needed

**Comment by michaelryanmcneill** [comment]
This is weird....going to try again for you @jboxman-rh 

/cherry-pick rhacs-docs-3.72

**Comment by jboxman-rh** [comment]
I guess this needs to be manually cherry picked?

**Comment by michaelryanmcneill** [comment]
@jboxman-rh can you help manually cherry pick it?

**Pull request merged**